src/Compilers/CSharp/Test/Semantic/Semantics/UsingDeclarationTests.cs

```diff
@@ -369,7 +369,7 @@ public void UsingVariableCanBeInitializedWithExistingDisposable()
         {
             var source = @"
 using System;
-class C2
+class C2 : IDisposable
```

Comment:
Consider cloning this test (`UsingVariableCanBeInitializedWithExistingDisposable`) and making a ref struct version.